queries.sql

```diff
@@ -19,4 +19,70 @@ SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+ALTER TABLE animals ADD species varchar(255);
+
+
```

Comment:
Well done altering the table properly, as requested by the previous reviewer. 👏🏾 💯 

This line of code should be in the `schema.sql` file. Note that everything that creates or change the tables structure should be saved in the schema.sql file.

1. Kindly move the SQL command on line 25 to its relevant schema file and your project will be ready for approval.